api-docs/list-clients.md

```diff
@@ -0,0 +1,17 @@
+#list-clients API
+#POST Request
+#URL /api/list-clients/
+#Parameters: {firstName, LastName}
+Return a list of people's first name, last name, date of birth, zip code, phone number added by and added date.
+
+#Result: An array of results
```

Comment:
This should be a paginated API, since there will be hundreds/thousands of responses.